packages/design-system/src/components/checkbox/stories/index.js

```diff
@@ -32,6 +30,19 @@ import { DarkThemeProvider } from '../../../storybookUtils';
 export default {
   title: 'DesignSystem/Components/Checkbox',
   component: Checkbox,
+  args: {
+    checkboxLabel1: 'Normal',
+    checkboxLabel2: 'Disabled',
+    checkboxLabel3: 'Checked',
+    checkboxLabel4: 'Checked and Disabled',
+  },
+  argTypes: {
+    checkboxLabel1: { name: 'Label 1' },
+    checkboxLabel2: { name: 'Label 2' },
+    checkboxLabel3: { name: 'Label 3' },
+    checkboxLabel4: { name: 'Label 4' },
+    handleChange: { action: 'checkbox changed' },
```

Comment:
This works so well!

💭 nit: we probably don't need a different label for each checkbox. Should they all share the same one? Doesn't matter though and seems annoying to update.

Reply:
Yeah. I went back and forth on whether to to make them all work as one or separate them out. I guess I decided on separate. heheh.